cumulusci/robotframework/tests/salesforce/pageobjects/objectmanager.robot

```diff
@@ -0,0 +1,74 @@
+*** Settings ***
+
+Resource        cumulusci/robotframework/Salesforce.robot
+Library         cumulusci.robotframework.PageObjects
+Suite Setup     Run keywords  Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Keywords ***
+Create Customfield In Object Manager
```

Comment:
```suggestion
Create Custom Field In Object Manager
```

(and then the lines which use the keyword also need to be updated)

Reply:
@rjanjanam don't you also need to update lines 41, 54, and 67 which use this keyword? Actually it's weird that the tests passed -- @boakley can you make sure they are included in CI?

Reply:
The tests are running -- I guess the keyword is found even though it is spelled with different whitespace. I'll fix the spaces and then approve this.